psycopg/pyproject.toml

```diff
@@ -13,8 +13,7 @@ classifiers = [
     "Operating System :: MacOS :: MacOS X",
     "Operating System :: Microsoft :: Windows",
     "Operating System :: POSIX",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3 :: Only",
```

Comment:
https://pypi.org/classifiers/ - just claiming no more python 2 and since Py3 is the only supportable version, it may be fine unless Py4 is issued and both to be maintained.

I can remove this `Only` claim if you like it better.

Reply:
I don't know the semantic difference between `3` and `3 :: Only`. I know that `3` doesn't mean `3.0`, so for me it had the same meaning as "not 2".

I am ok with this change, no problem. I just wanted to know if you had a better "official explanation".